datasets/coco.py

```diff
@@ -13,6 +13,7 @@
 frPyObjects = _mask.frPyObjects
 
 BASEDIR = pathlib.Path(__file__).parent.parent / "datasets/COCO"
```

Comment:
can you change this to `BASEDIR = pathlib.Path(__file__).parent.parent / "datasets" / "COCO"`?
it will work for those running on windows

Reply:
Done.